I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 64, in script
    call screen crimeFox
ScriptError: could not find label 'continueFox'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\Raissa\Downloads\renpy-7.4.5-sdk/renpy/bootstrap.py", line 326, in bootstrap
    renpy.main.main()
  File "C:\Users\Raissa\Downloads\renpy-7.4.5-sdk/renpy/main.py", line 617, in main
    run(restart)
  File "C:\Users\Raissa\Downloads\renpy-7.4.5-sdk/renpy/main.py", line 148, in run
    renpy.execution.run_context(True)
  File "C:\Users\Raissa\Downloads\renpy-7.4.5-sdk/renpy/execution.py", line 922, in run_context
    context.run()
  File "game/script.rpy", line 64, in script
    call screen crimeFox
  File "C:\Users\Raissa\Downloads\renpy-7.4.5-sdk/renpy/script.py", line 910, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'continueFox'.

Windows-10-10.0.18362
Ren'Py 7.4.5.1648
Delphina e Fox 1.0
Sun Aug 22 16:00:55 2021
